stingray/simulator/simulator.py

```diff
@@ -311,10 +311,10 @@ def periodogram(self, lc, seg_size=None):
         -------
         ps: numpy.ndarray
             The array of normalized squared absolute values of Fourier
-            amplitudes    
+            amplitudes
 
         """
         if seg_size is None:
             seg_size = self.lc.tseg
 
-        return AveragedPowerspectrum(lc, seg_size).ps
+        return AveragedPowerspectrum(lc, seg_size).power
```

Comment:
It's just that one line, right? Please do; I'll send a PR as soon as I've figured out what is currently going wrong in the simulator.